### ISSUE TITLE
postcss-color-mix proposal

### ISSUE THREAD

**Issue by iamstarkov** [jonathantneal/precss#43]
I developed that plugin to have ability to mix colors in css like I used to have in Sass. will you be interested to have `color-mix` in `precss`?

http://sass-lang.com/documentation/Sass/Script/Functions.html#mix-instance_method

**Comment by stevenvachon** [comment]
Use [cssnext](https://npmjs.com/postcss-cssnext) as it supports the CSS4 standard `color(blend)`.

**Comment by jonathantneal** [comment]
FWIW, for years now `color()` has not been a standard — it is `color-mod()`. PreCSS uses it via PostCSS Preset Env which uses https://github.com/jonathantneal/postcss-color-mod-function

**Comment by AntonioRedondo** [comment]
Hi @jonathantneal. I'm bumping up the PreCSS version from 2.0.0 to 4.0.0 on an outdated web project and I see `postcss-color-function` isn't included any more since version 3.0.0. I also see the `color-mod()` function is disabled by default when using PostCSS Preset Env through PreCSS. How can we enable this function when it's used through PreCSS? Is [this wiki page](https://github.com/jonathantneal/precss/wiki/Options) up to date?

Perhaps something like:
```javascript
postcss({
  precss({
    features: {
      'color-mod-function': { unresolved: 'warn' }
  }),
  // other plug-ins...
})
```

Update: the above code seems to work!
